src/instructionForm.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+export default function InstructionForm(props) {
+  const [instructionInput, setInstructionInput] = useState('');
+    
+  function handleSubmit(e) {
+    e.preventDefault();
+    props.setInstructions([...props.instructions, instructionInput]);
+  }
+
```

Comment:
Going forward lets have all state and handler functions all live in app.js and pass down with props